.gitignore

```diff
@@ -6,4 +6,6 @@ syntax:glob
 *db
 #*
 arc_pass.txt
-captcha.png
\ No newline at end of file
+captcha.png
+
+.DS_Store
```

Comment:
Please keep .DS_Store in your own global .gitignore so that we can keep this .gitignore project-specific.